lint.swift

```diff
@@ -0,0 +1,547 @@
+            ["before": "\n- Fixed a bug.   \n  Issue:", "after": "\n- Fixed a bug.  \n  Issue:"],
+            ["before": "\n- Fixed a bug !\n  Issue:", "after": "\n- Fixed a bug !  \n  Issue:"],
+            ["before": "\n- Fixed a bug ! \n  Issue:", "after": "\n- Fixed a bug !  \n  Issue:"],
+            ["before": "\n- Fixed a bug !  \n  Issue:", "after": "\n- Fixed a bug !  \n  Issue:"],
+        ]
+    )
+
+    // MARK: ChangelogEntryLeadingWhitespaces
+    try Lint.checkFileContents(
+        checkInfo: "ChangelogEntryLeadingWhitespaces: The links line of a Changelog entry should start with two whitespaces.",
+        regex: #"\n( {0,1}| {3,})(Tasks?:|Issues?:|PRs?:|Authors?:)"#,
+        matchingExamples: ["\n- Fixed a bug.\nIssue: [Link](#)", "\n- Fixed a bug. \nIssue: [Link](#)", "\n- Fixed a bug.    \nIssue: [Link](#)"],
+        nonMatchingExamples: ["- Fixed a bug.\n  Issue: [Link](#)"],
+        includeFilters: [changelogFile],
+        autoCorrectReplacement: "\n  $2",
+        autoCorrectExamples: [
+            ["before": "\n- Fixed a bug.\nIssue: [Link](#)", "after": "\n- Fixed a bug.\n  Issue: [Link](#)"],
+            ["before": "\n- Fixed a bug.\n Issue: [Link](#)", "after": "\n- Fixed a bug.\n  Issue: [Link](#)"],
+            ["before": "\n- Fixed a bug.\n    Issue: [Link](#)", "after": "\n- Fixed a bug.\n  Issue: [Link](#)"],
+        ]
+    )
+
+    // MARK: ClosureParamsParantheses
+    try Lint.checkFileContents(
+        checkInfo: "ClosureParamsParantheses: Don't use parantheses around non-typed parameters in a closure.",
+        regex: #"\{\s*\(((?!self)[^):]+)\)\s*in"#,
+        matchingExamples: ["run { (a) in", "run { (a, b) in", "run { (a, b, c) in"],
+        nonMatchingExamples: ["run { (a: Int) in", "run { (a: Int, b: Int) in", "run { (a, b) -> String in"],
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles],
+        autoCorrectReplacement: #"{ $1 in"#,
+        autoCorrectExamples: [
+            ["before": "run { (a) in", "after": "run { a in"],
+            ["before": "run { (a, b) in", "after": "run { a, b in"],
+            ["before": "run { (a, b, c) in", "after": "run { a, b, c in"]
+        ]
+    )
+
+    // MARK: EmptyMethodBody
+    try Lint.checkFileContents(
+        checkInfo: "EmptyMethodBody: Don't use whitespace or newlines for the body of empty methods.",
+        regex: ["declaration": #"(init|func [^\(\s]+)\([^{}]*\)"#, "spacing": #"\s*"#, "body": #"\{\s+\}"#],
+        matchingExamples: [
+            "init() { }",
+            "init() {\n\n}",
+            "init(\n    x: Int,\n    y: Int\n) { }",
+            "func foo2bar()  { }",
+            "func foo2bar(x: Int, y: Int)  { }",
+            "func foo2bar(\n    x: Int,\n    y: Int\n) {\n    \n}"
+        ],
+        nonMatchingExamples: ["init() { /* comment */ }", "init() {}", "func foo2bar() {}", "func foo2bar(x: Int, y: Int) {}"],
```

Comment:
Codacy found an issue: [Don't keep empty methods in code without commenting inside why they are needed or a `// TODO: not yet implemented`.](https://app.codacy.com/gh/Flinesoft/HandySwift/pullRequest?prid=5730214)